.github/workflows/test_docker.yml

```diff
@@ -0,0 +1,58 @@
+name: Test Docker
+on:
+  push:
+    paths:
+      - '.github/workflows/test_docker.yml'
+      - 'driver/**'
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '.github/workflows/test_docker.yml'
+      - 'driver/**'
+    branches:
+      - master
+jobs:
+  build:
+    timeout-minutes: 60
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Get Node PKG Cache dir
+      id: node-pkg-cache
+      run: |
+        node -e "const path = require('path'); const fs = require('fs'); const cachePath = path.join(os.tmpdir(), 'pkg-cache'); fs.mkdirSync(cachePath); console.log('::set-output name=dir::' + cachePath)"
```

Comment:
why not simply `mkdir -p /tmp/pkg-cache` ?

Reply:
We used it once it was cross-os compatible.

Reply:
Let's remove it then!